qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/ls/api/QuteTemplateCodeActionResolveProvider.java

```diff
@@ -17,13 +17,22 @@
 import com.redhat.qute.commons.GenerateMissingJavaMemberParams;
 
 /**
- * Request additional information from the associated java language server in
- * order to resolve the workspace edit to create a field, getter, or template
- * extension for a given type.
+ * Provides information from the associated java language server in order to
+ * resolve code actions.
  *
  * @author datho7561
  */
-public interface QuteTemplateGenerateMissingJavaMember {
+public interface QuteTemplateCodeActionResolveProvider {
```

Comment:
I find QuteTemplateCodeActionResolveProvider is difficult to understand that, it is a provider for textEdit to update Java file and not a code action

What about `QuteTemplateJavaTextEditProvider` ?